docs/reference/authorization/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Everything in Bifröst starts here. A user is required to be authorized at first. Bifröst supports types, you can choose from to adjust to your needs.
+---
+
+# Authorizations
+
+Everything in Bifröst starts here. A user is required to be authorized at first. Bifröst supports [different types of authorizations](#types), you can choose from to adjust to your needs.
```

Comment:
Everything in Bifröst starts here. A user must be authorized at first. Bifröst supports [different types of authorizations](#types) you can choose from according to your needs.